src/dists.ad.js

```diff
@@ -353,7 +365,10 @@ function multivariateGaussianScore(mu, cov, x) {
 
 var MultivariateGaussian = makeDistributionType({
   name: 'MultivariateGaussian',
-  params: [{name: 'mu', desc: 'mean vector'}, {name: 'cov', desc: 'covariance matrix'}],
+  desc: 'n-dimensional Gaussian.',
+  params: [{name: 'mu', desc: 'mean vector (array of reals)'},
+	   {name: 'cov', desc: 'covariance matrix  (array of array of reals' +
```

Comment:
There's a space missing at the end of this string (currently compiles to `realsthat`).